src/data/deleteMany.ts

```diff
@@ -17,11 +17,11 @@ const DeleteManySchema = Type.Union([
   DeleteManyByFilterSchema,
 ]);
 
-export type DeleteManyByVectorIdOptions = Static<
-  typeof DeleteManyByRecordIdSchema
->;
-export type DeleteManyByFilterOptions = Static<typeof DeleteManyByFilterSchema>;
-export type DeleteManyOptions = Static<typeof DeleteManySchema>;
+export type DeleteManyByVectorIdOptions = Array<string>;
+export type DeleteManyByFilterOptions = object;
+export type DeleteManyOptions =
+  | DeleteManyByVectorIdOptions
```

Comment:
I've never seen the `|` used on the first entry like this. Is that correct/valid syntax?

Reply:
This was a prettier addition, but yeah seems like valid syntax. I did have to look it up though, found this: https://github.com/prettier/prettier/issues/6299